xun/tests/test_stores.py

```diff
@@ -114,6 +116,8 @@ def test_store_is_collection(cls):
         assert store['c'] == 2
 
         assert set(store.keys()) >= {'a', 'b', 'c'}
+        if  hasattr(store.driver, 'cache'): store.driver.cache.close()
```

Comment:
This should not be needed, and if confusing

Reply:
Without additional closing of the DiskCache, the temporary cache database file (cache.db) is still taken and windows is failing on deleting the file.
Actually, this is exactly the part of the solution I'm unsure about.
And it's not confusing :) It just says to close cache if it's present in your driver :)

Reply:
DiskCache removed